versions/1.1/SPEC.md

```diff
@@ -0,0 +1,5846 @@
+
+#### Stripping Leading Whitespace
+
+When a command template is evaluate, the execution engine first strips out all *common leading whitespace*.
+
+For example, consider a task that calls the `python` interpreter with an in-line Python script:
+
+```wdl
+task heredoc {
+  input {
+    File infile
+  }
+
+  command<<<
+  python <<CODE
+    with open("~{in}") as fp:
+      for line in fp:
+        if not line.startswith('#'):
+          print(line.strip())
+  CODE
+  >>>
+  ....
+}
+```
+
+Given an `infile` value of `/path/to/file`, the execution engine will produce the following Bash script, which has removed the two spaces that were common to the beginning of each line:
+
+```sh
+python <<CODE
+  with open("/path/to/file") as fp:
+    for line in fp:
+      if not line.startswith('#'):
+        print(line.strip())
+CODE
+```
+
+If the user mixes tabs and spaces, the behavior is undefined. The execution engine should, at a minimum, issue a warning and leave the whitespace unmodified, though it may choose to raise an exception or to substitute e.g. 4 spaces per tab.
+
+### Task Outputs
+
+The `output` section contains declarations that are exposed as outputs of the task after a call to the task completes successfully. An output declaration must be initialized, and its value is evaluated only after the task's command completes successfully, enabling any files generated by the command to be used to determine its value.
+
+For example a task's `output` section might looks like this:
+
+```wdl
+output {
+  Int threshold = read_int("threshold.txt")
+  Array[File]+ csvs = glob("*.csv")
+  File? outfile = "path/to/outfile.txt"
+}
+```
+
+After the command is executed, the following outputs are expected to be found in the task execution directory:
+- A file called "threshold.txt", which contains one line that consists of only an integer and whitespace.
+- One or more files (as indicated by the `+` postfix quantifier) with the `.csv` extension in the working directory that are collected into an array by the [`glob`](#arrayfile-globstring) function.
+- There may or may not be an output file at `path/to/outfile.txt` (relative to the execution directory) - if the file does not exist, the value of `outfile` will be undefined.
+
+See the [WDL Value Serialization](#appendix-a-wdl-value-serialization-and-deserialization) section for more details.
+
+#### Files and Optional Outputs
+
+File outputs are represented as string paths.
+
+A common pattern is to use a placeholder in a string expression to construct a file name as a function of the task input. For example:
+
+```wdl
+task example {
+  input {
+    String prefix
+    File bam
+  }
+
+  command {
```

Comment:
If `command <<< >>>` is truly the preferred way, then the basic examples when you are not likely to run into bash variable confusion, should all have this format.  ONLY if the example would have issues due to variable definition formats should the `command { }` format be used. Perhaps a find/replace for all the command sections used in the doc might be semi-painless but worth it to set folks on The Good Path?

Reply:
done